docs/core/tools/csproj.md

```diff
@@ -69,6 +69,13 @@ This change does not modify the main mechanics of other includes. However, if yo
 ### Recommendation
 With csproj, we recommend that you remove the default globs from your project and only add file paths with globs for those artifacts that your app/library needs for various scenarios (runtime, NuGet packaging, etc.)
 
+## How to see the whole project as MSBuild sees it
+
+While those csproj changes greatly simplify project files, it is sometimes useful to be able to look at the fully expanded project as MSBuild sees it once the SDK and its targets are included. This can be done by simply typing the following in the command-line:
```

Comment:
"it is" :point_right: "it's"
There's a general leaning away from "you can" language. Also, "command-line" :point_right: "command prompt." Would you consider expanding a bit on ...

> This can be done by simply typing the following in the command-line:

... with something along the lines of this, which explains out a bit more what's going to happen? ....

> Preprocess the project with the following command and switch, which shows which files are imported, their sources, and their contributions to the build without actually building the project:

(:point_up: Shamelessly stolen sentiment from the MSDN topic.)